CHANGELOG.md

```diff
@@ -1,6 +1,13 @@
 Version History
 ---------------
 
+May 2017:
+
+v1.2.2 (patch release)
+- Fix seed-related bugs affecting wallet balance
+- Gracefully handle missing storage folders
+- Faster + smaller wallet database
```

Comment:
can you add a bullet point like "more than 5000 lines of testing, many bugfixes"

I don't know what the actual lines of testing is, but if you have an easy way to count, would be a good stat. It's not a small amount I know.